Core/Config.cpp

```diff
@@ -962,10 +962,10 @@ static ConfigSetting controlSettings[] = {
 	ConfigSetting("AnalogRotationCWKeyX", "AnalogRotationKeyCWY", "AnalogRotationKeyCWScale", "ShowAnalogRotationCWKey", &g_Config.touchAnalogRotationCWKey, defaultTouchPosHide, true, true),
 	ConfigSetting("AnalogRotationCCWKeyX", "AnalogRotationKeyCCWY", "AnalogRotationKeyCCWScale", "ShowAnalogRotationCCWKey", &g_Config.touchAnalogRotationCCWKey, defaultTouchPosHide, true, true),
 
-	ConfigSetting("AnalogDeadzone", &g_Config.fAnalogDeadzone, 0.0f, true, true),
+	ConfigSetting("AnalogDeadzone", &g_Config.fAnalogDeadzone, 0.15f, true, true),
 	ConfigSetting("AnalogInverseDeadzone", &g_Config.fAnalogInverseDeadzone, 0.0f, true, true),
-	ConfigSetting("AnalogSensitivity", &g_Config.fAnalogSensitivity, 1.0f, true, true),
-	ConfigSetting("AnalogIsCircular", &g_Config.bAnalogIsCircular, true, true, true),
+	ConfigSetting("AnalogSensitivity", &g_Config.fAnalogSensitivity, 1.1f, true, true),
+	ConfigSetting("AnalogIsCircular", &g_Config.bAnalogIsCircular, false , true, true),
```

Comment:
Isn't it circular on most platforms?  Won't this give a lot of people diagonal problems?

-[Unknown]

Reply:
It has a circular-ish range on many platforms, but the circular transform used here is a bit extreme and feels weird testing it with the new visualization. Generally it works better to just increase sensitivity (scale) and clamp to a square.

Reply:
Okay, got it.

-[Unknown]